src/runner.py

```diff
@@ -81,23 +81,17 @@ def _cleanup_paths(self):
         # ToDo: if stat != 0 then report error to API?
 
     def run(self):
-        sub_id = self._db.subscribe('node')
-        self._print("Listening for new checkout events")
+        sub_id = self._db.subscribe_node_channel(filters={
+            'op': 'updated',
+            'name': 'checkout',
+            'status': 'pass',
+        })
+        self._print("Listening for completed checkout events")
```

Comment:
as a follow up work we could get the `_print` method out of the class or even better switch to `logging` module.

Reply:
Yes, maybe create a GitHub issue for it?